sdc/datatypes/hpat_pandas_stringmethods_functions.py

```diff
@@ -1048,31 +1031,155 @@ def hpat_pandas_stringmethods_isdecimal_impl(self):
     return hpat_pandas_stringmethods_isdecimal_impl
 
 
+@sdc_overload_method(StringMethodsType, 'capitalize')
+def hpat_pandas_stringmethods_capitalize(self):
+    ty_checker = TypeChecker('Method capitalize().')
+    ty_checker.check(self, StringMethodsType)
+
+    def hpat_pandas_stringmethods_capitalize_impl(self):
+        item_count = len(self._data)
+        result = [''] * item_count
+        for idx in numba.prange(item_count):
```

Comment:
@AlexanderKalistratov Is it safe to modify numba list inside prange in this way?

Reply:
it is